index.html

```diff
@@ -20,25 +26,40 @@
     </header>
 
     <section>
-        <img src="Assets/Ellipse 11.png" class="deep_left">
-        <img src="Assets/Rectangle 4.png" class="light_right">
-        <img src="Assets/Ellipse 3.png" class="deep_right">
-        <img src="Assets/Rectangle 6.png" class="light_left">
-      <h1 class="intro">I'm Max <br> Glad to see you</h1>
-      
+      <img src="Assets/Ellipse8.png" class="small_bal" alt="small_ball_1"/>
+      <img src="Assets/Ellipse14.png" class="small_ball" alt="small_ball_2"/>
+      <img src="Assets/Ellipse5.png" class="ball" alt="ball"/>
+      <img src="Assets/Rectangle7.png" class="bar" alt="bar"/>
+      <img src="Assets/Ellipse11.png" class="deep_left" alt="left_ellipse"/>
+      <img src="Assets/Rectangle4.png" class="light_right" alt="right_box"/>
+      <img src="Assets/Ellipse3.png" class="deep_right" alt="right_ellispe"/>
+      <img src="Assets/Rectangle6.png" class="light_left" alt="left_box"/>
+      <h1 class="intro">
+        <strong>
```

Comment:
- [OPTIONAL] Kindly remove the `strong` tag for the `h1` text. This is because all headings in HTML already have that attribute, i.e, they are already bold by default.